dashboard/views.py

```diff
@@ -60,6 +60,27 @@ def gpa_map(grade):
 def get_home_template(request):
     return render(request, 'frontend/index.html')
 
+def view_names_mapping():
+    view_column_names: dict = {
+        'ap': CourseViewOption.show_assignment_planning.field.column,
+        'apv1': CourseViewOption.show_assignment_planning_v1.field.column,
+        'gd': CourseViewOption.show_grade_distribution.field.column,
+        'ra': CourseViewOption.show_resources_accessed.field.column
+    }
+    return view_column_names
+
+
+def get_course_view_options(is_admin, course):
+    view_column_names: dict = view_names_mapping()
+    global_views_disabled = []
+    for view in settings.VIEWS_DISABLED:
+        if view in view_column_names.values():
+            global_views_disabled.append((list(view_column_names.keys()))[list(view_column_names.values()).index(view)])
+    admin_course_views = CourseViewOption.objects.get(course=course).json(include_id=False)
+    course_view_options = {key: value for key, value in admin_course_views.items() if key not in global_views_disabled}
+    return admin_course_views if is_admin else course_view_options
```

Comment:
Wasn't sure what the requirement was here.  Admins are supposed to be able to enable the view even if it's been disabled?

Reply:
The main requirement is that instructors should not see the global disable option as select card. By default global disable option shown as disabled, they can toggle the option but it will be removed from instructor view. 

In Production, Admin's won't touch the course views options selected by the instructor. So That's the reason why i am just showing global disabled option for admin for previewing purpose

Reply:
@jennlove-um I am giving the option for enabling the option for the global disabled view for admins. Do Admins needs indication that this view is disabled globally. (Instructor won't be show those global disabled view)

Reply:
Jennifer told this is nice to have not required. So I have added this to issue [here](https://github.com/tl-its-umich-edu/my-learning-analytics/issues/1007)

Reply:
Alight, as long as it's understood that is how its going work it looks good to me.